app/assets/javascripts/components/overview/details.jsx

```diff
@@ -222,10 +222,10 @@ const Details = React.createClass({
         </div>
         <div className="module__data extra-line-height">
           {instructors}
-          {online}
-          {campus}
-          {staff}
-          {school}
+          {Features.wikiEd ? online : null}
+          {Features.wikiEd ? campus : null}
+          {Features.wikiEd ? staff : null}
```

Comment:
Let's keep the pattern already in place with `staff` where is defined (line 75), and put the conditionals there. That way, we only set up the components in the first place if they are needed.